src/lib/objects.js

```diff
@@ -0,0 +1,64 @@
+const createPerson = (name, age) => {
+  const person = {
+    name: name,
+    age: age
+  };
+  return person;
+};
+
+const getName = object => {
+  return object.name;
+};
+
+const getProperty = (property, object) => {
+  return object[property];
+};
+
+const hasProperty = (property, object) => {
+ return object.hasOwnProperty(property);
```

Comment:
When you are looking for return a boolean as you are here, you could use the logic you have here but with a bit of refactoring. If you were to first reduce this to a ternary:
```
  return object[property] ? true : false;
```
This does the same thing as you have above, but you could reduce this even further using a double negation:
```
  return !!object[property];
```
Much cleaner :)